src/lib/channels.py

```diff
@@ -24,7 +25,9 @@ class Channels(object):
         ("2019-04-23T00:00:00+0300", "2019-04-26T07:00:00+0300"),
         ("2019-04-30T00:00:00+0300", "2019-05-03T07:00:00+0300"),
         ("2019-05-07T00:00:00+0300", "2019-05-10T07:00:00+0300"),
-        ("2019-05-14T00:00:00+0300", "2019-05-17T07:00:00+0300")
+        ("2019-05-14T00:00:00+0300", "2019-05-17T07:00:00+0300"),
+        ("2019-05-21T00:00:00+0300", "2019-05-23T07:00:00+0300")
```

Comment:
Are these dates definitely correct? Seems to be one day shorter than usual

Reply:
That was an error! Thank you for noticing